src/main/java/it/gov/pagopa/pu/migration/service/migration/MigrationProcessingService.java

```diff
@@ -0,0 +1,119 @@
+package it.gov.pagopa.pu.migration.service.migration;
```

Comment:
This is a service dedicated to the activity, let's move accordingly to its package
```suggestion
package it.gov.pagopa.pu.migration.wf.service.ingestion;
```